apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -813,13 +815,13 @@ export const BaseNewsForm: React.FC<BaseNewsFormProps> = ({
                     }}
                     children={({ handleSave, isLoading, form }) => (
                         <>
-                            <Row gutter={BIG_VERTICAL_GUTTER}>
-                                <Col span={24}>
+                            <Row>
+                                <Col span={24} style={BIG_MARGIN_BOTTOM_STYLE}>
                                     <Row gutter={BIG_HORIZONTAL_GUTTER}>
                                         <Col span={formFieldsColSpan}>
-                                            <Row gutter={MEDIUM_VERTICAL_GUTTER}>
-                                                <Col span={24}>
-                                                    <Row>
+                                            <Row>
+                                                <Col span={24} style={ { marginBottom: '32px' } }>
```

Comment:
You already declared some style constants above. Maybe move this object to the constants too?